cli/cmd/initialize/init.go

```diff
@@ -0,0 +1,128 @@
+package initialize
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	"github.com/rilldata/rill/runtime/parser"
+	"github.com/spf13/cobra"
+)
+
+//go:embed templates/**/*.mdc
+var templateFS embed.FS
+
+func InitCmd(ch *cmdutil.Helper) *cobra.Command {
+	var olap, template string
+	var force bool
+
+	initCmd := &cobra.Command{
+		Use:   "init [<path>]",
+		Short: "Initialize Rill resources from templates",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+			targetPath := "."
+
+			if len(args) > 0 {
+				projectPath := filepath.Clean(args[0])
+				absPath, err := filepath.Abs(projectPath)
+				if err != nil {
+					return fmt.Errorf("failed to resolve path %q: %w", projectPath, err)
+				}
+				if err := os.MkdirAll(absPath, 0o755); err != nil {
+					return fmt.Errorf("failed to create directory %s: %w", absPath, err)
+				}
+				targetPath = absPath
```

Comment:
I think you the `Clean` and `MkdirAll` are not necessary, but on the other hand it needs `fileutil.ExpandHome` also, like here: https://github.com/rilldata/rill/blob/1c5e9fcad49ff603b03d1d19b10febf305cd40b4/cli/cmd/env/env.go#L29-L41